composer.json

```diff
@@ -64,5 +72,20 @@
     }
   ],
   "minimum-stability": "dev",
-  "prefer-stable": true
+  "prefer-stable": true,
+  "scripts": {
+    "post-install-cmd": [
+      "@prefix-dependencies"
+    ],
+    "post-update-cmd": [
+      "@prefix-dependencies"
+    ],
+    "prefix-dependencies": [
+      "php-scoper add-prefix --output-dir=./third-party --force --quiet",
```

Comment:
We're using `php-scoper add ...` - is that an alias or maybe something changed in a newer version?

Reply:
`add` is just a shorthand alias for `add-prefix` 👍